addOns/openapi/openapi.gradle.kts

```diff
@@ -24,8 +24,9 @@ configurations {
 }
 
 dependencies {
-    implementation("io.swagger:swagger-parser:1.0.33")
-    implementation("io.swagger:swagger-compat-spec-parser:1.0.33") {
+    implementation("org.glassfish.jaxb:jaxb-runtime:2.3.2")
```

Comment:
It can be a `compileOnly` dependency (this can also be amended to previous comment).